PBM-1102 Added a note about a confusing log message (#114)
percona/pbm-docs: docs/usage/restore.md

    After the restore is complete, do the following:

    1. Restart all `mongod` nodes. 

        !!! note

            You may see the following message in the `mongod` logs after the cluster restart:

            ```{.text .no-copy}
            "s":"I",  "c":"CONTROL",  "id":20712,   "ctx":"LogicalSessionCacheReap","msg":"Sessions collection is not set up; waiting until next sessions reap interval","attr":{"error":"NamespaceNotFound: config.system.sessions does not exist"}}}}
            ```

>             This is expected behavior of periodic checks of the database's ability to start. Ding the restore, the `config.system.sessions` collection is dropped but then Percona Server for MongoDB recreates it upon a cluster start. No action is required from your end.

Reviewer: ```suggestion
            This is expected behavior of periodic checks upon the database start. During the restore, the `config.system.sessions` collection is dropped but Percona Server for MongoDB recreates it eventually. It is a normal procedure. No action is required from your end.
```
I'd phrase it that way since I'm not sure that all of this happening during the start. Some of the logs may appear after the cluster is started. Plus typo "Ding->During"

    After the restore is complete, do the following:

    1. Restart all `mongod` nodes and `pbm-agents`. 

        !!! note

            You may see the following message in the `mongod` logs after the cluster restart:

            ```{.text .no-copy}
            "s":"I",  "c":"CONTROL",  "id":20712,   "ctx":"LogicalSessionCacheReap","msg":"Sessions collection is not set up; waiting until next sessions reap interval","attr":{"error":"NamespaceNotFound: config.system.sessions does not exist"}}}}
            ```

>             This is expected behavior of periodic checks of the database's ability to start. Ding the restore, the `config.system.sessions` collection is dropped but then Percona Server for MongoDB recreates it upon a cluster start. No action is required from your end.

Reviewer: ```suggestion
            This is expected behavior of periodic checks upon the database start. During the restore, the `config.system.sessions` collection is dropped but Percona Server for MongoDB recreates it eventually. It is a normal procedure. No action is required from your end.
```

Reviewer: same changes here

Author: Sorry, my bad:)
